functions/nft.test.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+const waitPort = require('wait-port')
+
+const params = {
+  port: 3001,
+  host: 'localhost',
+}
+
+beforeAll(async () => {
+  await waitPort(params)
+}, 60000)
```

Comment:
Instead of using `waitPort`, you can directly specify this in global-setup. Use the `port` option of `jest-dev-server`. This is a better option because it will delay _all_ tests, instead of just this test suite.